templates/blocks/happy_point.twig

```diff
@@ -12,16 +12,13 @@
 			</picture>
 			<div class="container">
 				<div class="row justify-content-md-center">
-					{% if ( (fields.mailing_list_iframe == 'true') or fields.iframe_url ) %}
+					{% if ( fields.mailing_list_iframe == 'true' or fields.iframe_url ) %}
 						{% if ( fields.iframe_url ) %}
 							{% set url = fields.iframe_url %}
 						{% else %}
 							{% set url = fields.engaging_network_id %}
 						{% endif %}
 						<div class="col-md-10 happy-point" id="happy-point" data-src="{{ url }}">
-							{% if ( fields.load_iframe ) %}
```

Comment:
Ops, It was my miss :( Thanks for spotting & fixing it. Initially I introduced the `load_iframe` param to load iframe Url in block editor. 
Does load iframe works in block editor?

Reply:
I push a fix to load iframe on block editor only. please have a look